src/msg/Message.h

```diff
@@ -553,7 +553,7 @@ ceph::ref_t<T> make_message(Args&&... args) {
 }
 }
 
-namespace crimson::net {
+namespace crimson {
```

Comment:
I think this commit should be combined with the one that converts all occurrences.  Otherwise we lose atomicity -- there would be a FTBFS between these two commits.